src/main/java/com/exadel/coolDesking/office/OfficeService.java

```diff
@@ -1,21 +1,43 @@
 package com.exadel.coolDesking.office;
 
+import com.exadel.coolDesking.exception.ConflictException;
+import com.exadel.coolDesking.exception.NotFoundException;
+import com.exadel.coolDesking.floorPlan.FloorPlanRepository;
 import lombok.RequiredArgsConstructor;
 import org.springframework.stereotype.Service;
 
+import java.util.List;
 import java.util.UUID;
 
 @RequiredArgsConstructor
 @Service
 public class OfficeService {
     private final OfficeRepository officeRepository;
+    private final FloorPlanRepository floorPlanRepository;
+    private final OfficeMapper officeMapper;
 
+    public List<Office> getAllOffices() {
+        return officeRepository.findAll();
+    }
 
-    public Office create(Office office) {
-        return null;
+    public String create(OfficeCreateDto officeCreateDto) {
+        if (officeRepository.existsByName(officeCreateDto.getName())){
+            throw new ConflictException("Office with this name already exists", Office.class, officeCreateDto.getName());
+        }
+        Office office = officeMapper.officeDtoToOffictEntity(officeCreateDto);
+        officeRepository.save(office);
+        return "OK";
```

Comment:
After you saved office entity, convert it to responseDto and return the Dto from this method

```suggestion
        return officeMapper.entityToDto(.....);
```